tests/test_skew_symmetric_matrices.py

```diff
@@ -0,0 +1,69 @@
+"""
+Unit tests for the skew symmetric matrices
+"""
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats.geometry.skew_symmetric_matrices import SkewSymmetricMatrices
+
+
+class TestSkewSymmetricMatrices(geomstats.tests.TestCase):
+    def setUp(self):
+        self.n_seq = [3, 4, 5, 6, 7, 8, 9, 10]
+        self.so = {n: SkewSymmetricMatrices(n=n) for n in self.n_seq}
```

Comment:
we should write `self.skew` because we can't make the distinction between SO and so, as uppercases letters are not permitted in the code if variables are not constants or class names